apps/pie-storybook/types/StoryOptions.ts

```diff
@@ -0,0 +1,20 @@
+type BackgroundColor = 'light (container-default)' |
+    'dark (container-dark)' |
+    'background-subtle' |
+    'brand orange' |
+    'saddlebrown' |
```

Comment:
nice

Reply:
I was so preoccupied with whether I could, I didn't stop to think if I should.

Reply:
I wouldn't approve the PR without this tbh